### PR TITLE
Develop front

### PR DESCRIPTION
**Pull request by Songsterr99**
Сверстал чаcть, которая видна незарегистрированному пользователю

### CODE REVIEW

File: front/package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "node-sass-chokidar": "^1.3.0",
+    "npm-run-all": "^4.1.3",
+    "react": "^16.4.1",
+    "react-bootstrap": "^0.32.1",
+    "react-dom": "^16.4.1",
+    "react-router-dom": "^4.3.1",
+    "react-scripts": "1.1.4"
+  },
+  "scripts": {
+    "build-css": "node-sass-chokidar src/ -o src/",
```
Comment (andrei-shenets-itechart) [inline front/package.json]: Why do you need separate build of sass?

File: front/public/index.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (andrei-shenets-itechart) [inline front/public/index.html]: Page can be not only in English in future

File: front/src/components/Footer/Footer.css [hunk 3]
```diff
@@ -0,0 +1,65 @@
+footer {
```
Comment (andrei-shenets-itechart) [inline front/src/components/Footer/Footer.css]: Unnecessary file in repository

File: front/src/components/Labs/Labs.js [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+import Lab from '../Lab/Lab';
+import './Labs.css';
+
+class Labs extends Component {
+
+  constructor(){
+    super();
+
+    this.state = {
+      //уберу в дальнейшем
```
Comment (andrei-shenets-itechart) [inline front/src/components/Labs/Labs.js]: At development time it is worth to exposrt such data to json file and use this file. Code will be cleaner

**Comment by andrei-shenets-itechart** [comment]
1. Branch conflicts with master
2. There is no correct .gitignore file in the project

File: front/src/components/JoinUsButton/JoinUsButton.scss [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+      @media only screen and (max-width: $break-small) {
+        display: none;
+      }
+    }
+
+    @media only screen and (max-width: $break-small) {
+      font-size: 20px;
+    }
+
```
Comment (AndreiShenets) [inline front/src/components/JoinUsButton/JoinUsButton.scss]: Please do not leave unnecessary empty strings and trailing white spaces

**Pull request merged**